velox/functions/prestosql/tests/BitwiseTest.cpp

```diff
@@ -120,6 +137,60 @@ TEST_F(BitwiseTest, bitwiseAnd) {
   EXPECT_EQ(bitwiseAnd<int64_t>(kMin64, -1), kMin64);
 }
 
+TEST_F(BitwiseTest, bitCount) {
+  EXPECT_EQ(bitCount(9, kMaxBits), 2);
+  EXPECT_EQ(bitCount(9, 8), 2);
+  EXPECT_EQ(bitCount(-7, kMaxBits), 62);
+  EXPECT_EQ(bitCount(-7, 8), 6);
+  EXPECT_EQ(bitCount(kMin64, kMaxBits), 1);
+  EXPECT_EQ(bitCount(kMax64, kMaxBits), 63);
+  EXPECT_EQ(bitCount(-2, 2), 1);
+
+  int64_t num = 7;
```

Comment:
This code is a bit hard to read. I wonder if it can be refactor for better readability.

```
auto assertInvalidBits = [] (int64_t num, int64_t bits) {
   assertUserError(
      [&]() { bitCount(num, bits); },
      fmt::format("Bits specified in bit_count must ....", bits));
};

assertInvalidBits(7, 1);
assertInvalidBits(7, 65);

auto assertNumberTooLarge = [] (int64_t num, int64_t bits) {
   assertUserError(
      [&]() { bitCount(num, bits); },
      fmt::format("Number must be representable ....", num, bits));
};

assertNumberTooLarge(7, 2);
assertNumberTooLarge(3, 2);
...
```

Reply:
Thanks for the suggestion. Refactored the code.